boot/boot.go

```diff
@@ -42,13 +42,14 @@ const (
 )
 
 // RebootInfo contains information about how to perform a reboot if
-// required
+// required.
 type RebootInfo struct {
-	// RebootRequired is true if we need to reboot after an update
+	// RebootRequired is true if we need to reboot after an update.
 	RebootRequired bool
-	// RebootBootloader will not be nil if the bootloader has something to say on
-	// how to perform the reboot
-	RebootBootloader bootloader.RebootBootloader
+	// BootloaderOptions can be provided if the bootloader has something
```

Comment:
relatedly this comment is not consistent with what with do, as we are setting this more often than described

Reply:
good point, i'll try to formulate this better